reflex-common/reflex-common/src/main/python/mcenter_client_v0/parallelm/mcenter_client/mcenter_client.py

```diff
@@ -1643,6 +1643,33 @@ def delete_source_control_comp(self, comp_id):
         r = self.delete(url, timeout=60)
         return r
 
+    def get_action_log(self, object_list=None, action_list=None, success=None, user_list=None, start_time=None, end_time=None):
```

Comment:
Add doc str to method.. with examples of parameters.

Reply:
Fixed